packages/lesswrong/components/posts/PostsPage/PostsPage.tsx

```diff
@@ -701,7 +741,7 @@ const PostsPage = ({fullPost, postPreload, eagerPostComments, refetch, classes}:
       classes.postBody,
       !showEmbeddedPlayer && classes.audioPlayerHidden
     )}>
-      {showSplashPageHeader && !commentId && !isDebateResponseLink && <h1 className={classes.secondSplashPageHeader}>
+      {showSplashPageHeader && !permalinkedCommentId && <h1 className={classes.secondSplashPageHeader}>
```

Comment:
I'm assuming `!commentId && !isDebateResponseLink` was here to check whether there is a permalinked comment at the top (although the previous condition didn't exactly match the one for showing the permalinked comment)